### PR TITLE
Badges and Buttons

### PR DESCRIPTION
**Pull request by praveenreddy1798**
**Badges**
contains Avatar Badge and Icon Badge

Screenshot 2022-02-08 at 6 47 39 PM

Screenshot 2022-02-08 at 6 47 48 PM

**Buttons**
contains Simple Buttons, Outline Buttons, Disabled Buttons, Link Buttons, Icon Buttons and Floating Button

buttons

### CODE REVIEW

**Review by TejasShekar — COMMENTED** [review]
Looks great. You can go ahead.

**Review by TejasShekar — APPROVED** [review]

**Review by TejasShekar — APPROVED** [review]

File: components/buttons/buttons.html [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+                <!-- Success Button -->
+                <button class="btn btn-success">Success</button>
+
+                <!-- Danger Button -->
+                <button class="btn btn-danger">Danger</button>
+
+                <!-- Warning Button -->
+                <button class="btn btn-warning">Warning</button>
+            </div>
+            <div class="flex-evenly mg-t-lg">
```
Comment (vaibhav18matere) [inline components/buttons/buttons.html:96]: can think of an alternative class name which is not so complicated

File: components/badges/badges.html [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+            <div class="flex-evenly mg-xlg">
+                <!-- Avatar Badge -->
+                <div class="badge-container avatar-badge">
+                    <img src="../../assets/profile.jpg" alt="avatar" class="avatar avatar-l" />
+                    <span class="rounded">1</span>
+                </div>
+            </div>
+            <div class="flex-evenly mg-t-lg">
+                <iframe src="https://carbon.now.sh/embed/a7ywQ74MK3MjdCbhkoxW"
+                    style="width: 640px; height: 200px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (vaibhav18matere) [inline components/badges/badges.html]: 1. can increase height so that vertical scroll will be removed from UI
2. can change to `width:100%`

**Review by vaibhav18matere — APPROVED** [review]

File: components/buttons/buttons.html (hunk 1 above)
Comment (praveenreddy1798) [reply components/buttons/buttons.html:96]: for which one ?

Comment (vaibhav18matere) [reply components/buttons/buttons.html:96]: `mg-t-lg`

**Pull request merged**